packages/panna-sdk/src/core/helpers/http.ts

```diff
@@ -124,31 +131,31 @@ export const request = async (
   // If cache exists, return the response from cache
   const cacheKey = JSON.stringify({ url, finalParams }, null, 0);
```

Comment:
Using `JSON.stringify` for cache keys can lead to inconsistent keys due to object property ordering.

Recommended fix:
```typescript
const cacheKey = `${url}:${JSON.stringify(finalParams, Object.keys(finalParams).sort())}`;
```